zwave_js_server/model/node/statistics.py

```diff
@@ -65,3 +82,34 @@ def commands_dropped_tx(self) -> int:
     def timeout_response(self) -> int:
         """Return number of Get-type cmds where node's response didn't come in time."""
         return self.data["timeoutResponse"]
+
+    @property
+    def rtt(self) -> Optional[int]:
+        """
+        Return average round trip time (RTT) to this node in milliseconds.
+
+        Consecutive measurements are combined using an exponential moving average.
+        """
+        return self.data.get("rtt")
+
+    @property
+    def rssi(self) -> Optional[Union[str, int]]:
+        """
+        Return average RSSI of frames received by this node.
+
+        Consecutive non-error measurements are combined using an exponential moving
+        average.
+        """
+        if (rssi := self.data.get("rssi")) is None:
+            return None
+        return friendly_rssi(rssi)
```

Comment:
I'd raise an exception with the RSSIError IntEnum as an attribute and the friendly message as exception argument, if there's an rssi error. That way we can make the typing more narrow and make the flow handling more clear for users.

Reply:
The only limitation with this approach is that if a client attempts to get `RouteStatistics.repeater_rssi` and any rssi value in the list is an error, the user won't be able to see the rest of the list due to this error. Do we feel that is acceptable?

Reply:
Maybe add the complete list with values as an attribute on the exception? That way it's clear that there's an error and the library user can still access the remaining data.

Reply:
Is this what you had in mind?

Reply:
Yes, great!